dev/scripts/requirements.txt

```diff
@@ -0,0 +1,3 @@
+pip==23.1.2
+requests==2.30.0
+beautifulsoup4=4.12.2
```

Comment:
where do we need beautifulsoup4?

Reply:
This is dep needed for another feature. Just removed it. Thanks!